pkg/apis/scheduling/types.go

```diff
@@ -196,6 +196,32 @@ type PodGroupSpec struct {
 	// NetworkTopology defines the NetworkTopology config, this field works in conjunction with network topology feature and hyperNode CRD.
 	// +optional
 	NetworkTopology *NetworkTopologySpec
+
+	// BunchPolicy defines policies for dividing all pods within the podGroup into multiple groups.
+	// +optional
+	BunchPolicy []BunchPolicySpec
```

Comment:
From my first view, this should be within the `NetworkTopology ` field. 

And the Bunch word seems not commonly used as Subgroup or Subset